Kernel/Bus/USB/USBPipe.cpp

```diff
@@ -94,6 +94,24 @@ ErrorOr<size_t> BulkInPipe::submit_bulk_in_transfer(size_t length, void* data)
     return transfer_length;
 }
 
+ErrorOr<size_t> BulkInPipe::submit_bulk_in_transfer(size_t length, UserOrKernelBuffer data)
+{
+    VERIFY(length <= m_dma_buffer->size());
+
+    MutexLocker lock(m_dma_buffer_lock);
+
+    size_t transfer_length = 0;
```

Comment:
Maybe just make it `size_t transfer_length = TRY(..);` below?

Reply:
I straight up copied the void* version, but sure